src/brev/produserbaredokumenter.js

```diff
@@ -9,19 +9,19 @@ const produserbaredokumenter = [
   },
   {
     kode: 'MELDING_FORVENTET_SAKSBEHANDLINGSTID_SOKNAD',
-    term: 'Melding om forventet saksbehandlingstid av søknad'
+    term: 'Melding om forventet sakbehandlingstid'
   },
   {
     kode: 'MELDING_FORVENTET_SAKSBEHANDLINGSTID_KLAGE',
-    term: 'Melding om forventet saksbehandlingstid av klage'
+    term: 'Melding om forventet sakbehandlingstid'
```

Comment:
Hvis dette er for visning i frontend, hvordan skiller vi med forrige?

Reply:
@frans-a 
Akkurat `Melding om forventet saksbehandlingstid` som her har fag bestemt at det egentlig ikke er nødvendig med søknad/klage i tittelen ettersom sakstype er synlig på en behandling. 
https://confluence.adeo.no/display/TEESSI/Kodeverk+i+Melosys?focusedCommentId=408094331#KodeverkiMelosys-DokumenteriMelosys

Den overordnede planen er som følger:
https://jira.adeo.no/browse/MELOSYS-4490# skal levere en mapping som tar over ansvaret som dokkat her per i dag.
Der blir det liggende følgende:

- malnavn i dokgen 
- dokumentkategori som benyttes i Joark
- dokumenttittel som benyttes i Joark 

Vi fortsetter å bruke vårt kodeverk med termer som vises i frontend, mens mappingen over benyttes for journalføring.